### ISSUE TITLE
should NotebookApp.secret_file be NotebookApp.secret_file

### ISSUE THREAD

**Issue by randy3k** [jupyter/notebook#1734]
The docs say this

<img width="750" alt="screen shot 2016-09-03 at 1 14 11 am" src="https://cloud.githubusercontent.com/assets/1690993/18222901/bfdfc302-7173-11e6-8a6f-d230727916eb.png">

should it be the following instead?
```
c. NotebookNotary.secret_file = "/path/to/notebook_secret"
```

Anyway, is there any way to specify it system-wise? Particularly, I want to use the same secret on jupyterhub so that users can freely exchange trusted notebooks.

**Comment by minrk** [comment]
@randy3k yes, that's correct. To do it system-wide (assuming it's a shared single computer), add the line to `/etc/jupyter/jupyter_notebook_config.py`:

```python
c. NotebookNotary.secret_file = "/path/to/notebook_secret"
```

Now, there is a **big** caveat to this that you should know before you do something like this:

> A shared secret file means that your users can produce notebooks that when other users *open* them, execute code immediately as the second user. This is the primary thing that trust protects users from.

**Comment by physinet** [comment]
This isn't working for me. I added the line to `~/.jupyter/jupyter_notebook_config.py` on two different computers. The line points to identical `notebook_secret` files on each computer. A notebook saved on one computer is not trusted by the other.

**Comment by physinet** [comment]
@minrk Could you test whether this works as intended?

**Comment by randy3k** [comment]
I also cannot get it to work.

**Comment by minrk** [comment]
@physinet on it!
